dashboard/app/assets/javascripts/soundEffects.js

```diff
@@ -204,6 +231,9 @@ Sound.prototype.preloadViaWebAudio = function (filename, onPreloadedCallback) {
   request.onload = function () {
     self.audioContext.decodeAudioData(request.response, function (buffer) {
       onPreloadedCallback(buffer);
+      if (self.config.playAfterRegister) {
```

Comment:
This might fit better in the caller's `onPreloadedCallback` argument inside of `play`

Could also potentially DRY up this segment:

```
if (this.config.playAfterRegister) {
      this.play(this.config.firstPlayOptions);
}
```

by extracting into an e.g. `Sound.prototype.onSoundLoaded` method